### PR TITLE
Add Aruba ClearPass devices

### PR DESCRIPTION
**Pull request by rtm516**
- ClearPass C1000 (JZ508A)
- ClearPass C2000 (JZ509A)
- ClearPass C2010 (R1V81A)
- ClearPass C3000 (JZ510A)
- ClearPass C3010 (R1V82A)

### CODE REVIEW

**Comment by rtm516** [comment]
Not stale

File: device-types/HPE/ClearPass-C1000.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+part_number: JZ508A
+u_height: 1
+is_full_depth: true
+front_image: true
+rear_image: true
+console-ports:
+  - name: Serial
+    type: rj-45
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ClearPass-C1000.yaml:14]: VGA is not a console port type. Please remove this

File: device-types/HPE/ClearPass-C1000.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: HPE
+model: ClearPass C1000
+slug: hpe-clearpass-c1000
+part_number: JZ508A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/HPE/ClearPass-C1000.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ClearPass-C2000.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+manufacturer: HPE
+model: ClearPass C2000
+slug: hpe-clearpass-c2000
+part_number: JZ509A
+u_height: 1
```
Comment (danner26) [inline device-types/HPE/ClearPass-C2000.yaml:6]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ClearPass-C2000.yaml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+is_full_depth: true
+front_image: true
+rear_image: true
+console-ports:
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ClearPass-C2000.yaml:12]: VGA is not a console port type. Please remove this

File: device-types/HPE/ClearPass-C2010.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+manufacturer: HPE
+model: ClearPass C2010
+slug: hpe-clearpass-c2010
+part_number: R1V81A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/HPE/ClearPass-C2010.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ClearPass-C2010.yaml [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ClearPass-C2010.yaml:12]: VGA is not a console port type. Please remove this

File: device-types/HPE/ClearPass-C3000.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+---
+manufacturer: HPE
+model: ClearPass C3000
+slug: hpe-clearpass-c3000
+part_number: JZ510A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/HPE/ClearPass-C3000.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ClearPass-C3000.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+front_image: true
+rear_image: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ClearPass-C3000.yaml:14]: VGA is not a console port type. Please remove this

File: device-types/HPE/ClearPass-C3010.yaml [hunk 9]
```diff
@@ -0,0 +1,27 @@
+---
+manufacturer: HPE
+model: ClearPass C3010
+slug: hpe-clearpass-c3010
+part_number: R1V82A
+u_height: 1
+is_full_depth: true
```
Comment (danner26) [inline device-types/HPE/ClearPass-C3010.yaml:7]: Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not
2. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

File: device-types/HPE/ClearPass-C3010.yaml [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: VGA
+    type: other
```
Comment (danner26) [inline device-types/HPE/ClearPass-C3010.yaml:12]: VGA is not a console port type. Please remove this

**Review by danner26 — CHANGES_REQUESTED** [review]

**Pull request merged**